### DISCUSSION TITLE
如何正确启用RSS

### DISCUSSION THREAD

**Discussion by 2061360308** [hugo-next/hugo-theme-next#151] [Q&A]
我在使用一款rss订阅器follow
我发现Hugo next的示例官网和我目前的情况一样无法被搜索到rss，但是项目主要贡献者他的个人网站是可以的，请问需要配置那些配置呢

参数如下
官网https://hugo-next.eu.org/
结果![Screenshot_20241230_012303.jpg](https://github.com/user-attachments/assets/d59713ea-ad18-4ec3-bcb1-01e0c247606b)

主要贡献者博客https://lisenhui.cn/
结果![Screenshot_20241230_012319.jpg](https://github.com/user-attachments/assets/2e57e516-6522-4403-8b1e-757e61cfd091)

**Comment by 2061360308** [comment]
已解决，需要自己在layout下面添加index.rss.xml模板内容可见官网，这里是我的记录https://2061360308.github.io/posts/%E6%9C%AC%E7%AB%99rss%E4%BF%AE%E5%A4%8D%E8%AE%B0%E5%BD%95/

**Comment by elkan1788** [comment]
@2061360308 这个情况有点奇怪的，`Hugo NexT`在设计时就是支持`rss.xml`文件生成，在示例的两个站点都有这两个文件的。

[https://preview.hugo-next.eu.org/rss.xml](https://preview.hugo-next.eu.org/rss.xml)
[https://hugo-next.eu.org/rss.xml](https://hugo-next.eu.org/rss.xml)

只需要在配置文件中打开rss的生成就可以，这个设置默认也是打开的，参考如下：

[config.yaml#L75](https://github.com/hugo-next/hugo-theme-next/blob/67b5ae3a3ab3d5da32121647e966548f446b290e/exampleSite/config.yaml#L75)

**Reply by 2061360308** [reply]
两个链接都可以正常解析，看来rss的生成没有问题，但是我之前使用的首页的链接去查找，显示找不到，会不会是演示官网没有在主页添加rss的链接，导致抓取不到，之后直接使用正确的rss链接就可以了
![Screenshot_20241231_075935.jpg](https://github.com/user-attachments/assets/1db60e69-a434-42ce-b7e8-132fbb41770a)

![Screenshot_20241231_080023.jpg](https://github.com/user-attachments/assets/b7628f7c-2dda-475b-89a6-b27e08d5214c)

**Reply by elkan1788** [reply]
或许有这个可能，这个软件有iphone版本吗？回头找时间试试看效果

**Reply by 2061360308** [reply]
https://github.com/RSSNext/Follow

直接使用在线版本也是可以的

**Reply by elkan1788** [reply]
> https://github.com/RSSNext/Follow
> 
> 直接使用在线版本也是可以的

好的，回头尝试下看看。
